src/06-loops.js

```diff
@@ -39,7 +52,15 @@ function rangeOdd(min, max) {}
  *  rangeEveryOther(11, 18);
  *  //> [ 11, 13, 15, 17 ]
  */
-function rangeEveryOther(min, max) {}
+function rangeEveryOther(min, max) {
+  
+  for (let i = min; i <= max; i++) {
```

Comment:
This code works but technically we are still checking every single element of the array.  Can you solve this to only check every other number - in essence taking less steps?